app/api/exports.py

```diff
@@ -20,8 +19,8 @@
 
 
 @export_routes.route('/events/<string:event_identifier>/export/json', methods=['POST'])
-@jwt_required
-def export_event(event_identifier):
+@is_coorganizer
+def export_event(event_id):
```

Comment:
If we even cast aside the fact that this will obviously not work and break stuff, how can you just replace event_identifier with event_id?

Where does it say that frontend is not using event_identifier to export events?
You can't just break compatibility like this

Reply:
@iamareebjamal Please check all file changes. You can find the logic in permissions.py. Everything is working and tested.

Reply:
There are 29 tests failing

Reply:
How can everything be working

Reply:
Secondly, the path variable meantions `<string:event_identifier>` and the argument is event ID...

Reply:
@iamareebjamal Those changes are done in permissions.py. Please have a look at that. I will look at the tests.

Reply:
I understand that. But this is very confusing. It should not be like that. There should be 1:1 mapping of expectation and reality. the argument names should match and conversion from event_identifier to event_id should not be done implicitly. If you want to do that, create another explicit decorator. Doing it in `is_coorganizer` is a violation of single responsibility principle

Reply:
@iamareebjamal Ohh, ok. On it.